scripts/devportal/update-themes-and-modules.sh

```diff
@@ -25,33 +30,53 @@ function add_all_repositories {
 function add_all_modules_repositories {
     for repo_name in ${modules[@]}
     do
-        git remote add -f ${repo_name} git@bitbucket.org:pronovix/${repo_name}.git
+      register_repository ${repo_name}
     done
 }
 
 function add_all_themes_repositories {
     for repo_name in ${themes[@]}
     do
-        git remote add -f ${repo_name} git@bitbucket.org:pronovix/${repo_name}.git
+      register_repository ${repo_name}
```

Comment:
See my above comment regarding modules; this is the same about themes this time.